fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,142 @@
+                            "prediction": '_',
+                        }
+                        agg_metrics_path = ['aggregated_metrics', ORG_ID, project["name"], model["id"]]
+                        print("ProjectModel: ", project["name"], model["id"], metric)
+                        result = client._call(agg_metrics_path, json_request)
+
+                        print("Agg_metrics_path: ", agg_metrics_path)
+                        print("json request: ", json_request, "\n")
+                        print("Result : ", result, "\n")
+
+                        # iterate through the json result for that specific metric
+                        for single_value in result["values"]:
+                            start_time = int(time.time()) * 1000
+
+                            # Every metric has a different way of providing the value. So handle them separetly.
+                            if metric == 'traffic_count':
+                                value = single_value["value"]
+                                print("Final list: ", project["name"], model["id"], start_time, metric, value)
+                                create_tags(project=project["name"], model=model["id"])
+                                self.gauge(metric, value, tags)
+
+                            elif metric == 'output_average' or metric == 'integrity_violation_count':
+                                for key, value in single_value["value"].items():
+                                    new_metric = key
+                                    value = value
+                                    print("Final list: ", project["name"], model["id"], start_time, new_metric, value)
+                                    create_tags(project=project["name"], model=model["id"])
+                                    self.gauge(metric, value, tags)
+
+                            elif metric == 'histogram_drift':
+                                for key, value in single_value["value"].items():
+                                    new_metric = "histogram_drift-" + key
+                                    value = value
+                                    print("Final list: ", project["name"], model["id"], start_time, new_metric, value)
+                                    create_tags(project=project["name"], model=model["id"], feature=key)
+                                    self.gauge(metric, value, tags)
+
+                            elif metric == 'feature_average':
+                                for key, value in single_value["value"].items():
+                                    new_metric = key
+                                    value = value
+                                    print("Final list: ", project["name"], model["id"], start_time, new_metric, value)
+                                    create_tags(project=project["name"], model=model["id"], feature=key)
+                                    self.gauge(metric, value, tags)
```

Comment:
Make sure to add `fiddler.feature_average` to metadata.csv since you are emitting this.